src/bindings/python/src/pyopenvino/graph/node_output.hpp

```diff
@@ -50,6 +50,15 @@ void regclass_graph_Output(py::module m, std::string typestring)
         return std::hash<VT*>()(self.get_node()) + self.get_index();
     });
 
+    output.def("__copy__", [](ov::Output<VT>& self) {
+        return ov::Output<VT>(self);
+    });
+
+    output.def("__deepcopy__", [](ov::Output<VT>& self, py::dict& memo) {
+        PyErr_SetString(PyExc_TypeError, "cannot copy 'openvino.runtime.ConstOutput' object.");
```

Comment:
Maybe this
```suggestion
        PyErr_SetString(PyExc_TypeError, "cannot deepcopy 'openvino.runtime.ConstOutput' object.");
```
would be more explicit and clear for the user? Using `deepcopy` is a very conscious choice, that way it will be obvious what is going on from the start. "Cannot copy" may suggest that there has been an error inside `deepcopy` (for example, its recursive behavior encountered a single, non-copyable object). What do you think?

Reply:
I agree, changed copy to deepcopy